### ISSUE TITLE
Hypoplastic facial nerve

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11771]
**Preferred term label:**
Hypoplastic facial nerve

**Synonyms**
CN VII

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0877215

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/621655

**Your nano-attribution (ORCID)**
